examples/lib/defaultHtmlStepUi.js

```diff
@@ -276,6 +276,21 @@ function DefaultHtmlStepUi(_sequencer, options) {
         });
     });
 
+    $stepAll('.color-picker-target').each(function(i, input) {
+      $(input)
+        .data('initValue', $(input).val())
```

Comment:
Isn't this the same code that is used for every input? If yes, you could just extend the old code, just add another selector string.
`$stepAll('{oldSelector} .color-picker-target')`

Reply:
No, this isn't the exact same code, this doesn't use focus() method which was causing this bug

Reply:
Oh.